data/json/professions.json

```diff
@@ -2895,5 +2895,34 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "ident": "bio_medic",
+    "name": "Bionic Surgeon",
+    "description": "As one of the top surgeons in the country, you were chosen for an augmentation program to expand the medical field. With your experties and augmentations, you can preform precise surgery with little assitance.",
```

Comment:
Last words should be "perform precise surgery with little assistance"
There should be one more space after the period between the sentences.